EU4ToVic3Tests/V3WorldTests/PoliticalManagerTests/CountryTests/CountryTests.cpp

```diff
@@ -281,6 +281,39 @@ TEST(V3World_CountryTests, PopCounterSumsSubStatePop)
 	EXPECT_EQ(1500, country.getPopCount());
 }
 
+TEST(V3World_CountryTests, CultureBreakdownSumsSubStateCultures)
+{
+	V3::Country country;
+	std::vector<std::shared_ptr<V3::SubState>> substates;
+
+	EXPECT_EQ(0, country.getPopCount());
+	EXPECT_EQ(0, V3::Country::getPopCount(substates));
+
+	auto sub0 = std::make_shared<V3::SubState>();
+	auto sub1 = std::make_shared<V3::SubState>();
+
+	V3::Pop pop0, pop1, pop2;
```

Comment:
It would be more clear (and fewer lines) to use the constructor:
```suggestion
    V3:Pop pop0("english", "", "", 1000);
    V3:Pop pop1("german", "", "", 500);
    V3:Pop pop1("english", "", "", 500);
```